commands/user_e2e_test.go

```diff
@@ -6,11 +6,12 @@ package commands
 import (
 	"fmt"
 
+	"github.com/hashicorp/go-multierror"
```

Comment:
The linter wants this a part of the last import block.

Reply:
fixed imports // make build which also runs the linter says things are fine now